### PR TITLE
feat: add event for the next meetup

### PR DESCRIPTION
**Pull request by astorije**
## PR Checklist

- [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/boston-ts-website/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

Looking forward to it!

@JoshuaKGoldberg would you mind creating the event on Eventbrite? 🙏
EDIT: Done

### CODE REVIEW

**Comment by astorije** [comment]
omg all these checks are a lot 🤣

File: app/data/events.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+	{
+		"date": "2024-08-29T18:00:00.000-04:00",
+		"link": "https://www.eventbrite.com/e/boston-ts-club-v-the-compiler-strikes-back-tickets-???",
```
Comment (JoshuaKGoldberg) [inline app/data/events.json:4]: Eventbrite isn't letting me log in 🙃 but as soon as it does I will post back!

File: app/data/events.json [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+		"location": "Microsoft NERD Center",
+		"time": "5:30pm - 8:30pm",
+		"topics": ["How to rent a car with a JSON Web Token", "OpenJS Foundation"]
```
Comment (JoshuaKGoldberg) [inline app/data/events.json:7]: Oh these are for September.

File: app/data/events.json (hunk 1 above)
Comment (JoshuaKGoldberg) [reply app/data/events.json:4]:
```suggestion
		"link": "https://www.eventbrite.com/e/boston-ts-club-v-the-compiler-strikes-back-supabase-tickets-962814643517?aff=oddtdtcreator",
```

File: app/data/events.json (hunk 2 above)
Comment (JoshuaKGoldberg) [reply app/data/events.json:7]:
```suggestion
		"topics": [
		  "Declarative Builds and Deployments with Nix",
		  "Supabase is Sweet",
		]
```

**Review by JoshuaKGoldberg — DISMISSED** [review]
Ok we have a link now! 🙌

File: app/data/events.json (hunk 1 above)
Comment (astorije) [reply app/data/events.json:4]: @JoshuaKGoldberg what's the `?aff=oddtdtcreator` part?

File: app/data/events.json (hunk 2 above)
Comment (astorije) [reply app/data/events.json:7]: 🤦

File: app/data/events.json (hunk 1 above)
Comment (JoshuaKGoldberg) [reply app/data/events.json:4]: Oh oops that shouldn't be there

Comment (JoshuaKGoldberg) [reply app/data/events.json:4]:
```suggestion
		"link": "https://www.eventbrite.com/e/boston-ts-club-v-the-compiler-strikes-back-supabase-tickets-962814643517",
```

**Review by JoshuaKGoldberg — APPROVED** [review]
🚢

**Pull request merged**